src/components/__tests__/Header.test.tsx

```diff
@@ -0,0 +1,15 @@
+import { render, screen } from '@testing-library/react';
+import Header from '../Header/Header';
+import { describe, it, expect } from 'vitest';
+
+describe('Header', () => {
+  it('renders without crashing', () => {
+    render(<Header />);
+  });
```

Comment:
there are no sense to create such test, if you create a test like second
second test checks rendering as well, so it's quite useless